common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -216,8 +216,14 @@ def _test_text_field(
                 self.assertEqual(self.account_settings_page.value_for_text_field(field_id), new_value)
 
     def _test_dropdown_field(
-            self, field_id, title, initial_value, new_values, success_message=SUCCESS_MESSAGE, reloads_on_save=False
-    ):
+            self,
```

Comment:
The original line was too long, and we don't like having multiple arguments on one line if we're already using a multi-line format.  Which line do you mean is "extra?"